.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Questions and Discussions
+    url: https://github.com/pyvista/pyvista/discussions
+    about: For general questions and discussions
```

Comment:
What does this affect exactly? "ISSUE_TEMPLATE" sounds a bit misleading if this is for Discussions/

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

I think this is valid here.  Effectively, if someone goes to raise an issue when they really want a discussion, they'll be able to pick from a menu of choices.  Practically speaking, it should help reduce the number of discussions in issues and redirect them to somewhere to have discussions instead.

Reply:
Currently we have a single "choice" of "Bug" when you try to open an issue.
![Screenshot 2021-06-20 at 22-03-33 pyvista pyvista](https://user-images.githubusercontent.com/17914410/122686798-725f6c80-d213-11eb-89cd-e0a3a5894333.png)

Will this create a new entry here? I'm asking because the template seems to be a new file, so it's not clear to me how this will interact with the existing template (assuming one already exists). Sorry for being obtuse, I just don't think I can preview this feature anywhere.

Reply:
Please check this.
> Inspired by [this template](https://github.com/executablebooks/jupyter-book/issues/new/choose).

Reply:
I see, so .md files map to issue templates, while config.yml items create links. Thanks, @tkoyama010!